tests/multi_ptr/multi_ptr_members_fp64.cpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for multi_ptr members tests with double data type.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "multi_ptr_members.h"
+
+#define TEST_NAME multi_ptr_members_fp64
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+using namespace multi_ptr_members;
+
+/** tests the api for explicit pointers
+ */
+class TEST_NAME : public util::test_base {
+ public:
+  /** return information about this test
+   */
+  void get_info(test_base::info &out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+  /** execute this test
+   */
+  void run(util::logger &log) override {
+    auto queue = util::get_cts_object::queue();
+    if (!queue.get_device().has(sycl::aspect::fp16)) {
```

Comment:
```suggestion
    if (!queue.get_device().has(sycl::aspect::fp64)) {
```

Reply:
I must have missed that, why is that better than checking for the aspect?

Reply:
I've forgotten to write it, sorry. 
Basically, `util::extensions::availability` does the same thing, but it will protect us from a lot of changes in the future if method `queue.get_device().has(..)` will change. We will just modify the `availability` function instead of changing every test, that requires the `fp16` or `fp64` operations.